pkg/store/sandbox/netns_windows.go

```diff
@@ -38,23 +38,24 @@ type NetNS struct {
 
 // NewNetNS creates a network namespace for the sandbox
 func NewNetNS() (*NetNS, error) {
-	// TODO: JTERRY75 - Plugin cni
-	b := make([]byte, 10)
-	rand.Read(b)
-	return &NetNS{path: string(b)}, nil
+	temp := hcn.HostComputeNamespace{}
+	hcnNamespace, err := temp.Create()
+	if err != nil {
+		return nil, err
+	}
+
+	return &NetNS{path: string(hcnNamespace.Id)}, nil
 }
 
 // LoadNetNS loads existing network namespace. It returns ErrClosedNetNS
 // if the network namespace has already been closed.
 func LoadNetNS(path string) (*NetNS, error) {
-	// TODO: JTERRY75 - Plugin cni
 	return &NetNS{restored: true, path: path}, nil
```

Comment:
Do you want to query hcn in this case to make sure that network ID is still valid?